.gitignore

```diff
@@ -2,3 +2,4 @@ sandbox
 go-wfs
 .vscode/
 vendor/
+setup
```

Comment:
How does this file get generated?

Reply:
That's just a convenience script I use to set things like GOPATH.